scss/_patterns_buttons.scss

```diff
@@ -1,70 +1,53 @@
-// Default blockquote styling
-@mixin vf-buttons {
-  @include vf-primary-button;
-  @include vf-secondary-button;
-}
+// Default button styling
+@mixin vf-p-buttons {
+
+  .button {
+    padding: .75rem 1.5rem;
 
-// Primary button styling
-@mixin vf-primary-button {
-  button,
-  input[type='button'],
-  input[type='reset'],
-  input[type='submit'],
-  .button--primary {
-    @include vf-button($color-light, $color-light-grey, null);
+    &--neutral {
+      @extend .button;
```

Comment:
Yes, compiled sass is as expected: http://www.sassmeister.com/gist/45bd0689c590c18597089cbc93597029

the reason why I didn't use a placehodler is becuase `.button` is the 'base' use-case which can then be modified to provide the other variants,